lnwire/lnwire.go

```diff
@@ -171,12 +171,35 @@ func writeElement(w io.Writer, element interface{}) error {
 			}
 		}
 	case *btcec.Signature:
+		// Serialize the signature with all the checks that entails
 		sig := e.Serialize()
-		if len(sig) > 73 {
-			return fmt.Errorf("Signature too long!")
+		var b [64]byte
+		// Extract lengths of R and S
```

Comment:
ACK. I've improved the comments around this with an explanation and the format of the serialized sig.